README.md

```diff
@@ -31,6 +31,8 @@ will return metrics for a HTTP probe against google.com. The `probe_success`
 metric indicates if the probe succeeded. Adding a `debug=true` parameter
 will return debug information for that probe.
 
+Note the `probe_module_unknown` metric indicates if the module used is unknown.
```

Comment:
This doesn't belong here, please remove it.

Reply:
Done 👍